zap/src/main/java/org/zaproxy/zap/view/MultipleRegexesOptionsPanel.java

```diff
@@ -76,6 +82,67 @@ public MultipleRegexesOptionsPanel(Dialog owner, RegexesTableModel model) {
 
         this.owner = owner;
         getTable().setSortOrder(0, SortOrder.ASCENDING);
+        JButton importButton =
+                new JButton(Constant.messages.getString("context.include.import.label"));
+        importButton.addActionListener(e -> importButtonClicked());
+        addButton(importButton);
+    }
+
+    /**
+     * Checks if {@code regex} is contained in {@code regexes} and if it is a valid regex pattern.
+     * If {@code regexes} is null, then the duplication check is omitted
+     *
+     * @param regex A regex as string
```

Comment:
`regex` → `trimmedRegex`, or rename the parameter to `regex` (which has more occurrences).

Reply:
fixed documentation